src/trace-forwarder/Makefile

```diff
@@ -6,7 +6,7 @@
 default: build
 
 build:
-	RUSTFLAGS="--deny warnings" cargo build -v
+	RUSTFLAGS="--deny warnings" cargo build --target x86_64-unknown-linux-musl --release -v
```

Comment:
Hi @Tim-Zhang. Could you use `uname -m` to avoid hard-coding the architecture maybe?

Reply:
thank you, will do that

Reply:
done